src/cursor.ts

```diff
@@ -330,7 +393,39 @@ export function defaultDecodeCursor<
     if (field[0] !== expectedName) {
       throw new Error("Unexpected field name");
     }
+
+    if (field[1] === "null") {
+      field[1] = null;
+    }
   }
 
   return Object.fromEntries(parsed) as DecodedCursor<DB, TB, O, T>;
 }
+
+const minMaxValues: Record<SimpleColumnDataType, { min: any; max: any }> = {
+  varchar: { min: "", max: "\uffff" },
+  char: { min: "", max: "\uffff" },
+  text: { min: "", max: "\uffff" },
+  integer: { min: -2147483648, max: 2147483647 },
+  boolean: { min: false, max: true },
+  "double precision": { min: -1.7e308, max: 1.7e308 },
+  decimal: { min: "-Infinity", max: "Infinity" },
+  numeric: { min: "-Infinity", max: "Infinity" },
+  date: { min: "0001-01-01", max: "9999-12-31" },
+  datetime: { min: "0001-01-01 00:00:00", max: "9999-12-31 23:59:59" },
+  time: { min: "00:00:00", max: "23:59:59" },
+  timetz: { min: "00:00:00+00", max: "23:59:59+14" },
+  timestamp: { min: "0001-01-01 00:00:00", max: "9999-12-31 23:59:59" },
+  timestamptz: { min: "0001-01-01 00:00:00+00", max: "9999-12-31 23:59:59+00" },
+};
```

Comment:
This isn't perfect but it seems to be the most correct way of handling nullable field pagination (we have something similar currently in our bespoke non-lib pagination)

Reply:
Not sure if we want tests for all of these cases perhaps?